packages/augur-ui/config/webpack.common.config.js

```diff
@@ -58,17 +61,18 @@ module.exports = {
         test: /\.tsx?$/,
         loader: "ts-loader",
         options: {
+          experimentalWatchApi: true,
```

Comment:
Note: This makes webpack use the same watch api as tsc, which works a lot better.